server/config/config.go

```diff
@@ -9,9 +9,12 @@ import (
 )
 
 type ServerSettings struct {
-	SiteURL                 string
-	ListenAddress           string
-	KnowledgeGraphImportURL string
+	SiteURL                         string
+	ListenAddress                   string
+	KnowledgeGraphImportURL         string
+	SessionIdleTimeoutInMinutes     int
+	ExtendSessionLengthWithActivity bool
+	SessionLengthInMinutes          int
```

Comment:
If I understand correctly, SessionLengthInMinutes refers to the session length regardless of activity, is that correct?
If that is the case, should we implement session revocation when the session length limit is exceeded?